cowprotocol/composable-cow
# Add Sepolia deployments

# Description

This PR records the deployment of the composable cow contracts on Sepolia.

To guarantee that the contracts deploy to the same deterministic addresses work, we needed to restore the remapping to those used in earlier deployments: remappings are included in the Solidity metadata, which change the bytecode and in turn the addresses.

# Changes

- Sepolia deployment
- Update readme with new contracts and information on how to deploy

## How to test

Check verified contract code on Etherscan and compare the addresses in the readme with those in the `run-latest.json`.
